File: java/18_multiples_of_a_number.java
import java.io.File;
import java.io.IOException;
import java.util.Scanner;

public class Main
{
	public static void main(String args[])
	{
		File file = new File(args[0]);

		try
		{
			Scanner scanner = new Scanner(file);
			while (scanner.hasNextLine())
			{
				String line = scanner.nextLine();
				if (line.equals(""))
					continue;

				String tokens[] = line.split(",");
				int x = Integer.parseInt(tokens[0]);
				int n = Integer.parseInt(tokens[1]);

				int v;
				for (; v < x; v += n);

				System.out.println(v);
			}

			scanner.close();
		}
		catch (IOException e)
		{
		}
	}
}

File: javascript/18_multiples_of_a_number.js
var fs = require('fs');
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
	if (line == '') {
		return;
	}

	var tokens = line.split(',');
	var x = parseInt(tokens[0]);
	var n = parseInt(tokens[1]);

	var v = n;
	for (; v < x; v += n);

	console.log(v);
});

